src/components/structures/RoomView.js

```diff
@@ -916,9 +916,14 @@ module.exports = React.createClass({
 
         ContentMessages.sendContentToRoom(
             file, this.state.room.roomId, MatrixClientPeg.get(),
-        ).catch((error) => {
+        ).done(() => {
```

Comment:
"any unhandled rejection that ends up here will ... be thrown as an error" (see http://bluebirdjs.com/docs/api/done.html)

this should be catch(...).then(...)